packages/config/src/proxy.js

```diff
@@ -165,6 +166,9 @@ function createInsightsProxy({
                     changeOrigin: true,
                     autoRewrite: true,
                     ws: true,
+                    onProxyReq: (...args) => {
+                        cookieTransform(...args);
```

Comment:
Once consumer uses `proxyRoutes` they can't really change what are the custom entitlements. I would call `redirect.onProxyReq` if it is a function.
```suggestion
                        typeof redirect.onProxyReq === 'function' ? redirect.onProxyReq(...args) : cookieTransform(...args);
```